packages/storage/exp/index.ts

```diff
@@ -44,6 +52,16 @@ export {
  */
 const STORAGE_TYPE = 'storage-exp';
 
+export function getStorage(app: FirebaseApp): StorageService {
+  // Dependencies
```

Comment:
This comment confuses me more than it helps :)

Reply:
I think that's a copy/paste artifact! Removed.